asylum-connect/control-panel
# Feature Control Panel Latino to Latinx and Native Two-Spirit Updates

## Description
Changed in properties.json:
 "community-latino" to "community-latinx" 
 "community-native-american" to "community-native-american-two-spirit"

*NOTE: this PR requires a change in the catalog as well as a DB migration
migration - https://github.com/asylum-connect/inreach-api/pull/179
catalog - https://github.com/asylum-connect/inreach-catalog/pull/331

## Asana ticket:
https://app.asana.com/0/1202317426899416/1202588495993669

asana tickets for the catalog and DB migration changes:
https://app.asana.com/0/1202317426899416/1202692325395625
https://app.asana.com/0/1202317426899416/1202775476231973

## PR Checklist




## How to Test
hop into control panel and check for community-latinx and community-native-american-two-spirit under the properties drop down, the change won't fully be reflected in the resources until after the database migration step is completed
